modules/ROOT/pages/markdown.adoc

```diff
@@ -0,0 +1,91 @@
+= Markdown plugin
+:navtitle: Markdown
+:description: The Markdown Premium plugin detects pure markdown from a paste event within a TinyMCE editor instance.
+:description_short: Convert pure markdown on paste.
+:keywords: plugin, Markdown.
+:pluginname: Markdown
+:plugincode: markdown
+:plugincategory: premium
+
+include::partial$misc/admon-premium-plugin.adoc[]
+
+include::partial$misc/admon-requires-7.0v.adoc[]
+
+The {pluginname} Premium plugin detects pure markdown from a paste event within a {productname} editor instance.
+
+== Using the {pluginname} plugin
+
+
+=== Setup
```

Comment:
Empty section. Setup documented below.

Reply:
Noted: we usually cleanup the anything now required before merging.

Reply:
```suggestion
```